src/utils/object.js

```diff
@@ -21,6 +21,10 @@ export const evaluate = (template, model) => {
   }
 };
 
+export const isEvaluateParam = param => {
+  return /.*\{\{.+\}\}.*/.test(param);
```

Comment:
I don't think this should be standalone util.
I would place this function inside `buildDynamicURL`

Reply:
I think it can be standalone because it can be used in other places and not just inside of `buildDynamicURL`

Reply:
where?

Reply:
We don't have it, but we may want to send some dynamic parameters to the back end in the feature.
I intend that this function can stand by itself without `buildDynamicURL.`
WDYT?

Reply:
I don't see any future use to this. In case we'll have, let's consider it. In the meanwhile, I don't see it much generic.

Reply:
Not only this, I see you're using it only once in `buildDynamicURL`, so I don't even see it necessary as a function at all.